ChildMindInstitute/mindlogger-app-refactor
# feature/aadding privacy policy to app, creating a helper to call OpeU…

### 📝 Description

🔗 [Jira Ticket M2-8061](https://mindlogger.atlassian.net/browse/M2-8061)
#### This PR includes the following updates:
* Added Privacy Policy links to the Sign In, Sign Up, and About pages.
* Added Terms of Service link to the About page.
* Improved handling of line breaks for the Greek language.
* Created a helper function to streamline link sharing across components.
* Additionally, some minor fixes were made to enhance overall functionality.

### 📸 Screenshots
signIn:
<img width="325" alt="Screenshot 2024-10-23 at 2 56 32 PM" src="https://github.com/user-attachments/assets/dfb3e00f-90f7-4bf1-b3c2-0fb35cd27bd7">

signUp:
<img width="331" alt="Screenshot 2024-10-23 at 3 02 15 PM" src="https://github.com/user-attachments/assets/4eb075b6-5441-42be-b2f0-784e2f7d7ddb">

ENGLISH:
<img width="329" alt="Screenshot 2024-10-23 at 2 55 15 PM" src="https://github.com/user-attachments/assets/f4caca9d-d980-4546-ac1a-6bab3cc89cea">

GREEK BEFORE:
<img width="393" alt="Screenshot 2024-10-23 at 2 39 40 PM" src="https://github.com/user-attachments/assets/919b1648-5f30-486f-bc1e-65fca1eb26aa">

GREEK AFTER:
<img width="333" alt="Screenshot 2024-10-23 at 2 55 51 PM" src="https://github.com/user-attachments/assets/b397e5bc-4447-49d5-846d-d1802bc37b84">
<img width="336" alt="Screenshot 2024-10-23 at 2 56 03 PM" src="https://github.com/user-attachments/assets/36c43599-ab25-4749-ba3f-879f0b923893">

FRENCH:
<img width="326" alt="Screenshot 2024-10-23 at 3 03 02 PM" src="https://github.com/user-attachments/assets/2c06e04a-fd26-462a-a901-39d26d99da04">

### 🪤 Peer Testing

* lauch the app
#### to test in english: 
* navigate to signIn, signUp  and check the new privacy policy link ( I left some prints showing where its located )
* after the sigIn go to about screen clicking the button at the bottom of the first screen,  scroll down and check the new pricavy policy and terms of service link.
#### to test in other languages:
* go to user settings and change the language
* now perform the same test mentioned before in about page, signIn and SignUp 

### ✏️ Notes

Any question let me know, thanks.
